docs/intro/philosophy.rst

```diff
@@ -136,22 +136,22 @@ that captures the spirit of what DataLad is, and here is a brief overview on it.
    your files into something that only DataLad or a specialized tool can read.
    A PDF file (or any other type of
    file) stays a PDF file (or whatever other type of file it was)
-   whether it is managed by DataLad or not. This guarantees that users will not loose
-   data or data access if DataLad would vanish from their system, or even when DataLad
-   would vanish from the face of Earth. Using DataLad thus does not require or generate
+   whether it is managed by DataLad or not. This guarantees that users will not lose
+   data or access if DataLad would vanish from their system (or from the face of the 
+   Earth). Using DataLad thus does not require or generate
    data structures that can only be used or read with DataLad -- DataLad does not
    tie you down, it liberates you.
 
 #. Furthermore, DataLad is developed for **complete decentralization**.
    There is no required central server or service necessary to use DataLad. In this
-   way, no central infrastructure needs to be maintained (or paid for) --
-   your own laptop is the perfect place to live for your DataLad project, as is your
-   institutions webserver, or any other common computational infrastructure you
+   way, no central infrastructure needs to be maintained (or paid for).
+   Your own laptop is the perfect place for your DataLad project to live, as is your
+   institution's webserver, or any other common computational infrastructure you
    might be using.
 
 #. Simultaneously, though, DataLad aims to
    **maximize the (re-)use of existing 3rd-party data resources and infrastructure**.
-   Users *can* use existing central infrastructure should they want to.
+   Users *can* use an existing central infrastructure should they want to.
```

Comment:
Sorry for being nit-picky, but I don't want to create the impression that *one* infrastructure needs to be chosen (any number would be fine)

```suggestion
   Users *can* use existing central infrastructure should they want to.
```

Reply:
In that case I'd suggest changing it to "Users *can* use existing central infrastructures should they want to." Would that fit better?

Reply:
sounds good!